### PR TITLE
Fix travis cache

### PR DESCRIPTION
**Pull request by Jackie-Jiang**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ before_install:
   - curl -sSL -o ~/bin/install-jdk.sh https://raw.githubusercontent.com/sormuras/bach/master/install-jdk.sh && chmod +x ~/bin/install-jdk.sh
   - source ./.travis/.travis_set_deploy_build_opts.sh
 
+before_cache:
+  - rm -rf $HOME/.m2/org/apache/pinot
```
Comment (haibow) [inline .travis.yml:15]: should it be `$HOME/.m2/repository/org/apache/pinot`?

Comment (Jackie-Jiang) [reply .travis.yml:15]: Yes, good catch

**Comment by fx19880617** [comment]
This is great!
Quick question, do we also need to do so for quickstart jobs?

**Comment by Jackie-Jiang** [comment]
> This is great!
> Quick question, do we also need to do so for quickstart jobs?

@fx19880617 The main purpose for this fix is to remove extra files before uploading cache (with the fix the tests finally passed). Seems the quick-start tests also work fine, so I'll leave it like this now.

**Pull request merged**